lib/runner/run.js

```diff
@@ -67,10 +67,13 @@ _.extend(Run.prototype, {
             return callback(new Error('run: already running'));
         }
 
-        _.isFinite(_.get(this.options, 'timeout.global')) &&
-            backpack.timeback(callback, this.options.timeout.global, this, function () {
```

Comment:
I mean, `backpack.timeback` removes the normalized functions from the `callback`, and that is later used for the `run.triggers`.